docs/integrations/databases/open-telemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+
+##Fields creation in Sumo Logic for Redis
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/), which will be created as part of Redis App install if not already present.
+
+**db.cluster.name** - User configured. Enter a name to identify this Redis cluster. This cluster name will be shown in the Sumo Logic dashboards.
+**db.system** - Has fixed value of redis
+**deployment.environment** - User configured. This is the deployment environment where the Redis cluster resides. For example: dev, prod or qa.
+**sumo.datasource** - has fixed value of redis
+
+##Prerequisite
+
+This section provides instructions for configuring log collection for Redis running on a non-Kubernetes environment for the Sumo Logic App for Redis. By default, Redis logs are stored in a log file.
+
+Follow the instructions to set up log collection:
+
+1. To configure the Redis log file, locate your local `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` configuration file in the database directory. By default, Redis logs are stored in `/var/log/redis/redis-server.log`.
+
+After determining the location of conf file modify the `redis.conf` configuration file logging parameters:
+
+  *   Open `redis.conf` configuration file in a text editor.
+  *   Set the following config parameters:
+    *   Specify the server verbosity level. The value loglevel in conf file can be set to one of the following:
+      *   debug (a lot of information, useful for development/testing)
+      *   verbose (many rarely useful info, but not a mess like the debug level)
+      *   notice (moderately verbose, what you want in production probably)
+      *   warning (only very important/critical messages are logged) loglevel notice
+    *   Specify the log file name. Also the empty string can be used to force Redis to log on the standard output. Note that if you use the standard output for logging but daemonize, logs will be sent to `/dev/null` logfile
+  *   Save the `redis.conf` file and restart the redis server using command : `sudo  service redis-server restart`
+
+Once the logs are configured to be written to a local file, follow the below step to configure collection in Sumo.
+
+##Collection Configuration & App installation
+
+As part of setting up the collection process and app installation user can select the App from App Catalog and click Install App. Please follow the steps below:
+
+### Step1: Set up Collector
+
+If you want to use an existing OpenTelemetry Collector, then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector, please select the "Add a new Collector" option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.
```

Comment:
```suggestion
This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
```